.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "8"
+  - "10"
```

Comment:
could just go with node_js: 10 I think

Reply:
In y opinion we should test on current LTS and stable.

Reply:
What for though? This module doesn't run in Node, it runs in Browsers - would make more sense to test on a couple different browsers, Node is just running the test runner.

Reply:
Sure, bit keep in mind that also zestrunner (see the issue at karmatic) may fail under a different Node.js version and contributors (developers) also need to run the tests.